clusters/lib/gatekeeper/application.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: gatekeeper
+  labels:
+    nerc.mghpcc.org/sync-policy: common
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/ocp-on-nerc/gatekeeper.git
+    targetRevision: HEAD
+    path: REPLACEME
```

```suggestion
    path: gatekeeper-system/overlays/nerc-ocp-prod
```

Sorry, I didn't see these `REPLACEME` lines on my first pass.

---

These get replaced by patches in the cluster `kustomization.yaml`, e.g.:

```
  - target:
      kind: Application
      name: gatekeeper
    patch: |
      - op: replace
        path: /spec/source/path
        value: gatekeeper-system/overlays/nerc-ocp-prod
```

Which results in:

```
apiVersion: argoproj.io/v1alpha1
kind: Application
metadata:
  labels:
    nerc.mghpcc.org/sync-policy: common
  name: gatekeeper-prod
spec:
  destination:
    name: nerc-ocp-prod
    namespace: gatekeeper-system
  project: default
  source:
    path: gatekeeper-system/overlays/nerc-ocp-prod
    repoURL: https://github.com/ocp-on-nerc/gatekeeper.git
    targetRevision: HEAD
  syncPolicy:
    automated:
      selfHeal: true
    syncOptions:
    - ApplyOutOfSyncOnly=true
```

---

Ah, that makes sense, thanks for the info!